### PR TITLE
test/214 unit tests for UpdateTimelineItemHandler

### PR DESCRIPTION
**Pull request by yaninavoda**

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    {
+        _mockRepositoryWrapper = new Mock<IRepositoryWrapper>();
+        _mockMapper = new Mock<IMapper>();
+        _mockLogger = new Mock<ILoggerService>();
+    }
+
+    [Theory]
+    [InlineData("")]
+    [InlineData("Title")]
+    [InlineData("somthing else")]
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs:38]: something

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    }
+
+    private static UpdateTimelineItemRequestDto GetRequestDto(string? context)
+    {
+        return new UpdateTimelineItemRequestDto(
+            Id: 1,
+            StreetcodeId: 1,
+            Title: "Title",
+            Description: "Description",
+            Date: new DateTime(2024, 4, 11, 0, 0, 0),
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs:201]: Can we avoid warnings? 
image

**Review by OlhaPivniuk — COMMENTED** [review]
Didn't add Tests for UpdateTimelineItemRequestDtoValidator

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs (hunk 1 above)
Comment (yaninavoda) [reply Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs]: fixed typo

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs (hunk 2 above)
Comment (yaninavoda) [reply Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/TimelineItem/UpdateTimelineItemHandlerTests.cs]: added DateTimeKind

**Review by OlhaPivniuk — APPROVED** [review]

**Pull request merged**